README.md

```diff
@@ -103,8 +103,43 @@ fn fetch_an_integer() -> String {
 }
 ```
 
+## JSON Support
+
+Support for the RedisJSON Module can be enabled by specifying "json" as a feature in your Cargo.toml.
+
+`redis = { version = "0.17.0", features = ["json"] }`
+
+Then you can simply include the `JsonCommands` which will add the `json` commands to all Redis Connections (not to be confused with just `Commands` which only adds the default commands)
+
+```rust
+use redis::Client;
+use redis::JsonCommands;
+use redis::RedisResult;
+
+// Result returns Ok(true) if the value was set
+// Result returns Err(e) if there was an error with the server itself OR serde_json was unable to serialize the boolean
+fn set_json_bool<P: ToString>(key: P, path: P, b: bool) -> RedisResult<bool> {
```

Comment:
You make a very good point, I'll update it to say `ToRedisArgs` since I keep forgetting that in theory the entire Shrek movie as an mp4 file could be a redis key (it's binary safe)